src/Compilers/CSharp/Portable/Symbols/Source/SourceCustomEventAccessorSymbol.cs

```diff
@@ -68,7 +68,7 @@ internal sealed class SourceCustomEventAccessorSymbol : SourceEventAccessorSymbo
             }
             else
             {
-                var bodyOpt = syntax.Body;
+                var bodyOpt = (CSharpSyntaxNode)syntax.Body ?? syntax.ExpressionBody;
```

Comment:
>var bodyOpt = (CSharpSyntaxNode)syntax.Body ?? syntax.ExpressionBody; [](start = 16, length = 69)

Consider simply inlining ```syntax.Body != null || syntax.ExpressionBody != null``` into the ```if``` condition below and getting rid of the local.